examples/custom-ipld-formats/daemon-node.js

```diff
@@ -0,0 +1,96 @@
+// ordinarily we'd open a PR against the multicodec module to get our
+// codec number added but since we're just testing we shim our new
+// codec into the base-table.json file - this has to be done
+// before requiring other modules as the int table will become read-only
+const codecName = 'dag-test'
+const codecNumber = 392091
+
+const baseTable = require('multicodec/src/base-table.json')
+baseTable[codecName] = codecNumber
+
+// now require modules as usual
+const IPFSDaemon = require('ipfs-cli/src/daemon')
+const multihashing = require('multihashing-async')
+const multihash = multihashing.multihash
+const multicodec = require('multicodec')
+const CID = require('cids')
+const ipfsHttpClient = require('ipfs-http-client')
+const uint8ArrayToString = require('uint8arrays/to-string')
+
+async function main () {
+  // see https://github.com/ipld/interface-ipld-format for the interface definition
+  const format = {
+    codec: codecNumber,
+    defaultHashAlg: multicodec.SHA2_256,
+    util: {
+      serialize (data) {
+        return Buffer.from(JSON.stringify(data))
```

Comment:
Should we use `uint8ArrayFromString` here per [ipld/interface-ipld-format#utilserializeipldnode](https://github.com/ipld/interface-ipld-format#utilserializeipldnode). It also needs to be required

Reply:
It's not necessary because the daemon only runs under node and node Buffers are Uint8Arrays so the interface contract is fulfilled.